src/app/components/category-products/category-products.component.ts

```diff
@@ -95,7 +95,7 @@ export class CategoryProductsComponent {
     this._productsService.getAll(),
     this.category,
     this.sortBy$,
-    this.searchForm.valueChanges.pipe(startWith({ priceFrom: -1, priceTo: 9999999 })),
+    this.searchForm.valueChanges.pipe(startWith({ priceFrom: -1 })),
```

Comment:
czemu właściwie muszę podać cokolwiek w startWith? jesli usune tez i priceFrom to przestaje mi sie wyswietlac lista

to dlatego że w 94 linijce w category-products.components.ts mam combineLatest i musi się wygenerować jakakolwiek wartość dla searchForm, żeby dostarczyło wszystkie elementy do combineLatest i przeszło dalej?

Reply:
Dokładnie tak, bez tego dane załadowałyby się dopiero po zmianach w formularzu np. dodaniu priceFrom